pkg/types/openstack/machinepool.go

```diff
@@ -23,8 +23,8 @@ type MachinePool struct {
 	// +optional
 	AdditionalSecurityGroupIDs []string `json:"additionalSecurityGroupIDs,omitempty"`
 
-	// Zones contains a list of availabilty zones that instances can be deployed on
-	//
+	// Zones is the list of availability zones where the instances should be deployed.
+	// Uses Nova default availabilityZone by default
```

Comment:
This sentence is confusing... Maybe something like:
"If no zones are provided, all instances will be deployed in OpenStack Nova default availability zone."